### PR TITLE
Created toolbar and headline section

### PR DESCRIPTION
**Pull request by addisalemseifu**
-added index.html and style.css files with the proper tags, styles, and contents.
-created README.md file to elaborate the project well.
-added .gitignore file to ignore the node_modules files.
-configured linters.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# PortofolioMobileVersionRep
\ No newline at end of file
+
+## PortofolioMobileVersionRep
+> This is the first portofolio done depending on "Templet 1" of figma website.
+
```
Comment (DanteAlonsoHT) [inline README.md]: - [ ] You should add a better description, just try to explain what you did, what is the main scope, what kind of technologies or techniques you used, and so on. (Add like 3 lines or more at least in your description in your README file😉).
- [ ] Remember to include a section where you can explain how to install/set up this repository in a local environment, explaining how to use git to clone a repository, and so on. Remember to do this easy to understand for beginners or non-technical people

File: index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            <div class="contact_us_wrapper">
+                <a class="icon" href="https://twitter.com/SeifuAddisalem"><i class="fab fa-twitter"></i></a>
+                <a class="icon" href="https://www.linkedin.com/in/addisalemseifu/"><i class="fab fa-linkedin-in"></i></a>
+                <a class="icon" href="https://github.com/addisalemseifu"><i class="fab fa-github-square"></i></a>
+                
+                
+            </div>
+        </div>
+    </section>
+</body>
```
Comment (DanteAlonsoHT) [inline index.html]: - [ ] [OPTIONAL] Remember the mean goal in this project, in a professional environment the client or company will request you almost an exact copy from design, try to apply this mindset in this project (if you apply responsive design would be much better) For this reason try to make the CSS style more similar only applying some details as you can see in the blue color in the image below. Like font weight, font size, size, and so on.
- [ ] Remove the y-scrollbar in your title 😉 
- [ ] Add more space between your logo - menu logo and your first SVG sometimes your menu logo and your first SVG overlap each other when I change the screen size.

The Figma design | Your design

image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            <h1 class="headline">I'm Addisalem Glad to see you!</h1>
+            <p class="description">I am a software developer! I can help you build a product, feature or website look through some of my work and experiance! if you like what you see or have a project you need coded don't hesitate to contact me. </p>
+            <h2 class="connect">LETS CONNECT</h2>
+            <div class="contact_us_wrapper">
+                <a class="icon" href="https://twitter.com/SeifuAddisalem"><i class="fab fa-twitter"></i></a>
+                <a class="icon" href="https://www.linkedin.com/in/addisalemseifu/"><i class="fab fa-linkedin-in"></i></a>
+                <a class="icon" href="https://github.com/addisalemseifu"><i class="fab fa-github-square"></i></a>
+                
+                
+            </div>
```
Comment (DanteAlonsoHT) [inline index.html]: - [ ] Remove the empty lines. [35 ... 37].
- [ ] The `ul` HTML tag and `li` elements inside `ul` are missing and these are mandatory requirements, please add them, check the next example:
```
<ul class="contact_us_wrapper">
  <li><a class="icon" href="https://twitter.com/SeifuAddisalem"><i class="fab fa-twitter"></i></a></li>
  <li><a class="icon" href="https://www.linkedin.com/in/addisalemseifu/"><i class="fab fa-linkedin-in"></i></a></li>
  <li><a class="icon" href="https://github.com/addisalemseifu"><i class="fab fa-github-square"></i></a></li>
</ul>
```

- [ ] Probably your design will change after adding the previous requirement, please fix your styles to make your design similar to the Figma design you selected if this situation happens.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+            <div class="menue_bar">
+                <div class="bar"></div>
+                <div class="bar"></div>
+                <div class="bar"></div>
+            </div>
+        </nav>
+        <div class="porto_wrapper">
+            <h1 class="headline">I'm Addisalem Glad to see you!</h1>
+            <p class="description">I am a software developer! I can help you build a product, feature or website look through some of my work and experiance! if you like what you see or have a project you need coded don't hesitate to contact me. </p>
+            <h2 class="connect">LETS CONNECT</h2>
```
Comment (DanteAlonsoHT) [inline index.html]: - [ ] This title "LETS CONNECT" should be inside `a` HTML tag due to you will use this link to go to another section in your portfolio 😉 For example:

```
<h2 class="connect"><a>LETS CONNECT</a></h2>
```

- [ ]  Probably your design will change after adding the previous requirement, please fix your styles to make your design similar to the Figma design you selected if this situation happens.

**Review by DanteAlonsoHT — CHANGES_REQUESTED** [review]
<h1> Status: Required Changes 🚫 </h1>

Hi @addisalemseifu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ZeenatLawal — CHANGES_REQUESTED** [review]
Hi @addisalemseifu, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
- [ ] _Kindly check the comments above, made under the reviews of the previous code reviewer(s) and stating **not properly resolved**. Kindly take your time to correctly review those issues outlined by the previous code reviewers, and please check to see that the changes are working as intended._

## Optional suggestions
- I would suggest you use Grammarly to check typos on your README file to make sure there are no typos and your sentences are structured correctly.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (ZeenatLawal) [reply README.md]: - The second issue here (installation/setup steps) has not been properly resolved.

File: index.html (hunk 2 above)
Comment (ZeenatLawal) [reply index.html]: - The third issue here (space between menu logo and SVG) has not been properly resolved.

Comment (addisalemseifu) [reply index.html]: sorry but I did not understand what the SVG is?

**Review by afanimartin — APPROVED** [review]
Hi @addisalemseifu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### [OPTIONAL]
- It would be nice to format your readme professionally. Please refer to [this](https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) article


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**